components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -129,7 +129,7 @@ public void execute() {
             }
             String toolkitConfigPath = CmdUtils.getMainConfigLocation();
             init(projectName, toolkitConfigPath, deploymentConfigPath);
-            outStream.print("Generating sources...");
+            outStream.print("Generating sources... \n");
```

Comment:
This doesn't look good. This makes the `[DONE]` printed in next line which is not professional at all IMO. Earlier we had few difficulties in printing this in one line but I fixed it recently in one of my PRs. If we need to print warn messages then the best option is to keep all messages in a list and print those after the build command execution, in the `Main.java`

Reply:
If any of the options doesn't work my opinion is we should remove the half baked`Generating sources` message completely from the build output.